src/App.tsx

```diff
@@ -21,11 +37,13 @@ export const App: React.FC = () => {
             data-cy="perPageSelector"
             id="perPageSelector"
             className="form-control"
+            value={perPage}
+            onChange={onSelect}
           >
-            <option value="3">3</option>
-            <option value="5">5</option>
-            <option value="10">10</option>
-            <option value="20">20</option>
+            <option value={3}>3</option>
```

Comment:
Instead of hardcoding the options in the `select` element, you can create an array of options and map over it. This will make the code more readable and easier to maintain. Example: `const options = [3, 5, 10, 20]; options.map(option => <option value={option}>{option}</option>)`